io.github.mrquantumoff.mcmodpackmanager.yml

```
@@ -17,7 +17,7 @@ modules:
   - name: mcmodpackmanager
     buildsystem: simple
     build-commands:
-      - ar -vox minecraft-modpack-manager_2.4.3_amd64.deb
+      - ar -vox minecraft-modpack-manager_2.5.1_amd64.deb
```

```suggestion
      - ar -vox minecraft-modpack-manager_*_amd64.deb
```

this would save from having to update it.